manifests/f/Freenet/Freenet/1497/Freenet.Freenet.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Freenet.Freenet
+PackageVersion: "1497"
+PackageLocale: en-US
+Publisher: freenetproject.org

```

```suggestion
Publisher: https://www.hyphanet.org/index.html
```
This URL redirects for me. Do you get the same? Should we link directly to the redirect?